gprofiler/main.py

```diff
@@ -769,7 +785,6 @@ def parse_cmd_args() -> configargparse.Namespace:
     )
 
     args = parser.parse_args()
```

Comment:
I think you can just check `if args.subcommand == extract-resources` then call `copy_resources` then `return`. This way, we don't need to make any other changes in the `main` function